### PR TITLE
about/for-proposals: add "for proposals" text focusing on data

### PR DESCRIPTION
**Pull request by rkdarst**
I was asked to make a sample text that can be used for funding
proposals which are data-focused.  We have a lot to offer here.

- This is a draft and I don't think it's very good yet - please help
  revise.  I think they are missing most of the interesting things we
  do, as well as just not being very organized.

- I also copied the "acknowledging Triton" part here - it doesn't
  exactly go here but is somewhat related.

### CODE REVIEW

**Comment by eglerean** [comment]
I will try to comment on this tomorrow. I have some pre-made bits also for personal/sensitive data.

**Comment by rkdarst** [comment]
I don't see the changes... are they supposed to be added as suggestions?

File: about/for-proposals.rst [hunk 1]
```diff
@@ -28,5 +28,35 @@ e.g. University dedicated OpenStack based cloud resources and access
...
+is managed well.  Data storage includes 2PB of high-performance Lustre
+filesystem space connected directly to a computing cluster for
+efficient and secure analysis, and 1PB of reliable, backed-up storage
+space for longer-term storage.  Expert staff, both technical and
+administrative, provide advice and hands-on support in data storage,
+computation, FAIR principles, data management planning, as well as
+computation.
+
+Data management is designed with a focus on security.  Recommended
+storage locations are centrally located for security.  All data is
```
Comment (eglerean) [inline about/for-proposals.rst:52]:
```suggestion
storage locations are centrally located for security.  Computing nodes and data storage servers are physically located at CSC, Keilaranta 14, Espoo. The server room is certified security level 3 (VAHTI-3) i.e. only authorized personnel with clearance are given access to it and there is continuous camera surveillance. All data is
```

**Comment by rkdarst** [comment]
I think this is ready, so I am merging it.

**Pull request merged**